modules/hal_nordic/nrfx/Kconfig

```diff
@@ -7,6 +7,12 @@ config HAS_NRFX
 menu "nrfx drivers"
 	depends on HAS_NRFX
 
+config NRFX_FORCE_ALT
+	string "nrfx source directory"
+	default ""
+	help
+	  Allows the user to overwrite the default nrfx directory with an alternative location.
```

Comment:
Can you give me a very good reason as to why you need this to be a Kconfig and why you can't use a custom west manifest for this?

Reply:
This would allow the team responsible for developing nrfx to test it against Zephyr before releasing the new revision. The nrfx is developed outside of hal_nordic so a custom west manifest would require a fork of hal_nordic somewhere and some mechanism to copy developed nrfx to it which is cumbersome.

The proposed solution requires no overhead for CI - just one additional repo to clone and one additional Kconfig called when invoking Twister. This way we could continuously test nrfx changes instead of testing them after releasing the new revision and prepare Zephyr patches beforehand.

Reply:
A similar thing is done for the clock control driver where the `CLOCK_CONTROL_NRF_FORCE_ALT` symbol allows an alternative implementation of the driver.